digdag-plugin-utils/src/main/java/io/digdag/util/Workspace.java

```diff
@@ -126,9 +151,12 @@ public String templateCommand(TemplateEngine templateEngine, Config params, Stri
                 catch (FileNotFoundException | NoSuchFileException ex) {
                     throw new ConfigException("File not found: " + ex.getMessage(), ex);
                 }
-                catch (RuntimeException | IOException ex) {
+                catch (IOException ex) {
                     throw new ConfigException("Failed to read a template file: " + command + ": " + ex.getClass(), ex);
                 }
+                catch (RuntimeException ex) {
+                    throw new ConfigException("" + ex.getMessage() + ": " + command, ex);
```

Comment:
This works when `ex.getMessage()` returns null...although String.format is preferred anyway